addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithDabControlXML.java

```diff
@@ -97,71 +99,71 @@ public void update() throws IOException, ReceivedMessageParseException {
             throw new ReceivedMessageParseException("<Play_Info>GetParam failed: " + response);
         }
 
-        //Sample response:
```

Comment:
This should be reverted, can you guard it using an `// @formatter:off ` and `// @formatter:on`